tests/cases/fourslash/linkedEditingJsxTag4.ts

```diff
@@ -0,0 +1,49 @@
+/// <reference path='fourslash.ts' />
+
+// for readability purposes
+//const jsx = (
+//   <div<T>>
+//      <p>
+//         <img />
+//      </p>
+//   </div>
+//);
+
+// @Filename: /typeTag.tsx
+////const jsx = (
+////   </*0*/div/*1*/</*2*/T/*3*/>/*4*/>/*5*/
+////      <p>
+////         <img />
+////      </p>
+////   <//*6*/div>
+////);
+
+// @Filename: /typeTagError.tsx
+////const jsx = (
+////   </*10*/div/*11*/</*12*/T/*13*/>/*14*/
+////      </*15*/p />
+////   <//*16*/div>
+////);
+
+const startPos = test.markerByName("0").position;
+const endPos =  test.markerByName("6").position;
+const linkedCursors = {
+    ranges: [{ start: startPos, length: 3 }, { start: endPos, length: 3 }],
+};
+
+verify.linkedEditing( {
+    "0": linkedCursors,
+    "1": linkedCursors,
+    "2": undefined,

```

Comment:
Maybe this is a lot of work, but maybe it's better to have a baseline for this instead, rather than all of these `undefined`s? I know that a lot of our current tests have this, but, I think we're trying to make them friendlier, e.g. @sandersn's new completion baselines.